stable/database/files/backup_hooks.py

```diff
@@ -153,22 +182,40 @@ def freeze_archive(unfreeze=False):
         subprocess.check_output(
             ["fsfreeze", action, ARCHIVE_DIR], stderr=subprocess.STDOUT
         )
-
+    else:
+        raise RuntimeError("not supported quiesce mode '{}'".format(FREEZE_MODE))
```

Comment:
Nit: Can you change this to "Unsupported quiesce mode..."?

Reply:
Good catch!